mesheryctl/cmd/root.go

```diff
@@ -82,14 +82,12 @@ var rootCmd = &cobra.Command{
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	fmt.Println("\n")
```

Comment:
Would you mind showing us how the effect of the code would be removed? Is there may make a non-new line situation? @kushthedude

Reply:
It was generating two blank lines which were not needed, as cursor was already printing on next line.